### PR TITLE
Merchant items index #18

### LINKED ISSUE #18: User Story 51, Merchant Items Index Page
```
As a merchant
When I visit my items page "/dashboard/items"
I see a link to add a new item to the system
I see each item I have already added to the system, including:
- the ID of the item
- the name of the item
- a thumbnail image for that item
- the price of that item
- my current inventory count for that item
- a link or button to edit the item

If no user has ever ordered this item, I see a link to delete the item
If the item is enabled, I see a button or link to disable the item
If the item is disabled, I see a button or link to enable the item
```

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #19

child of #79

depends on #32

### PR DESCRIPTION
**Pull request by allieIsOkay**
- Adds tests for merchant items index page ('/dashboard/items')
- Adds test for Item#ordered? instance method to determine if an item has been ordered
- Adds Item#ordered method
- Adds merchant items index page with basic bootstrap styling present
- Adds routing for merchant item crud actions except show
- Routing for specific enable and disable actions mirroring Admin routes used for item enabling and disabling routes.
closes #18

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -58,4 +58,8 @@ def average_fulfillment_time
       "Never been ordered"
     end
   end
+
+  def ordered?
+    order_items.count > 0
```
Comment (timnallen) [inline app/models/item.rb:63]: Great way to do this!

**Review by timnallen — APPROVED** [review]
Great work!

**Review by csvlewis — APPROVED** [review]
Nice work! Good job planning ahead with all of the bootstrap tags in your HTML.

**Review by teresa-m-knowles — APPROVED** [review]
Looks really good, thanks for looking ahead and doing the Bootstrap stuff from the get to
